pom.xml

```diff
@@ -93,12 +97,52 @@
                         <dependency>
                             <groupId>org.apache.maven.scm</groupId>
                             <artifactId>maven-scm-provider-gitexe</artifactId>
-                            <version>1.9.5</version>
+                            <version>2.1.0</version>
                         </dependency>
                     </dependencies>
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.sonatype.central</groupId>
+                <artifactId>central-publishing-maven-plugin</artifactId>
+                <version>0.7.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <publishingServerId>central</publishingServerId>
+                    <autoPublish>true</autoPublish>
+                    <skipPublishing>false</skipPublishing>
+                    <waitForPublishCompletion>true</waitForPublishCompletion>
+                    <waitUntil>validated</waitUntil>
```

Comment:
I'm assuming this should be `published` based on the `waitForPublishCompletion` above?